packages/paste-core/components/chat-log/src/ChatLogger.tsx

```diff
@@ -32,6 +34,12 @@ const ChatLogger: React.FC<ChatLoggerProps> = ({chats}) => {
     },
   });
 
+  React.useEffect(() => {
+    const chatItems = loggerRef.current?.querySelectorAll('[role="listitem"]');
+    const lastItem = chatItems?.[chatItems.length - 1];
+    lastItem?.scrollIntoView({behavior: 'smooth', block: 'nearest', inline: 'start'});
+  }, [chats]);
```

Comment:
Scroll into view new chat bubbles as they are added to the logger, open to better versions of this.

Reply:
This is fine for a demo. For more complex chat systems, people will likely want to implement a more robust scrollbar system.

Reply:
Hmm I wonder what they were referencing here https://github.com/facebook/lexical/pull/3378

Reply:
That's for the composer, this is for the chat log.